README.md

```diff
@@ -144,6 +144,13 @@ load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
 
 container_deps()
 
+load(
```

Comment:
container_deps above should call rules_docker_images

Reply:
Done